app/build.gradle

```diff
@@ -79,6 +80,6 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.0'
```

Comment:
3.5.1- the latest

Reply:
If we update this then we have to update the codelab. So I'm focusing mostly on BOM with the exception of other dependencies not mentioned in the codelab to reduce overhead.

Reply:
Ok, noted. Thanks!